Source/FullScreenPokemon.js

```diff
@@ -2228,6 +2248,86 @@ var FullScreenPokemon;
          * @param direction   The direction border being changed.
          */
         FullScreenPokemon.prototype.setThingBordering = function (thing, other, direction) {
+            if (thing.title === "Ledge") {
```

Comment:
Instead of hardcoding a reaction for ledges, a better way would add a `onBordering` hook (or some equivalent name) that Things could describe. Ledges could then declare it to be `setLedgeBordering`.